Explorer/Assets/DCL/Utilities/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace DCL.Utilities.Extensions
+{
+    public static class StringExtensions
+    {
+        public static bool IsValidUrl(this string url) =>
+            Uri.TryCreate(url, UriKind.Absolute, out Uri uriResult)
```

Comment:
`Uri` API is terrible: it's slow and allocating all over the place. That's why I didn't use it for `URLAddress` (etc). I know we don't have validation but if we want to have something it's better to implement it on our own or find something allocation-free and fast.